site/content/docs/user/ingress.md

```diff
@@ -112,6 +113,51 @@ Refer to [Using Ingress](#using-ingress) for a basic example usage.
 
 Additional information about Contour can be found at: [projectcontour.io](https://projectcontour.io)
 
+### Ingress Kong
+
+Deploy [Kong Ingress Controller (KIC)](https://docs.konghq.com/kubernetes-ingress-controller/2.1.x/concepts/design/).
+
+{{< codeFromInline lang="bash" >}}
+kubectl apply -f bit.ly/k4k8s
```

Comment:
bit.ly ? Don't we have a GitHub or corporate URL ?

Reply:
@aojea it's actually GH link https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/master/deploy/single/all-in-one-dbless.yaml.
Let me know if you want me to include a full link.

Reply:
If you don't mind? I think is most transparent

Reply:
sure thing! pushed an update in 17e8fa2